### DISCUSSION TITLE
Is horizontal scaling possible using the official helm chart

### DISCUSSION THREAD

**Discussion by abdennour** [YOURLS/YOURLS#3012] [Q&A]
The [official helm chart](https://github.com/YOURLS/charts/blob/master/charts/yourls/values.yaml) provides a way to horizontally scaling the app ( replicaCount).

Assuming that i am running Database externally, is Yourls a stateless app that can be horizontally scaling ?
What about Http Session of logged in user ? where it's stored ?  How session stickiness will be guaranteed with horizontal scaling

**Comment by ozh** [comment]
Out of the box there is no logged in user session storage. Authentication is done against stored cookie if present, or using provided credentials (see [here](https://github.com/YOURLS/YOURLS/blob/94f5db040555329b9f10652918a8ea0ca58339bb/includes/functions-auth.php#L24))

**Reply by LeoColomb** [reply]
I second @ozh, especially if you separate admin interface (you should not need to scale) from the redirect process (where there is zero auth, not even with cookie).
This separation is doable with a tweaked ingress controller but not shipped out of the box with the Helm chart.
